l1-contracts/contracts/state-transition/chain-deps/facets/Getters.sol

```diff
@@ -247,8 +248,9 @@ contract GettersFacet is ZKChainBase, IGetters, ILegacyGetters {
         return s.settlementLayer;
     }
 
-    function getDAValidatorPair() external view returns (address, address) {
-        return (s.l1DAValidator, s.l2DAValidator);
+    /// @inheritdoc IGetters
+    function getDAValidatorPair() external view returns (address, L2DACommitmentScheme) {
```

Comment:
External tooling might break, I'm not sure who is using this

Reply:
It might. It is breaking change anyway